### PR TITLE
add multiple rating functions at backend

### LINKED ISSUE #10: Add Skill score attribute in the Document_Review table
Add Skill score attribute (type: number) in the Document_Review table

### LINKED ISSUE #11: Add Experience score attribute in the Document_Review table
modify the new created java file DocumentReview.java

### LINKED ISSUE #13: Add GPA attribute in the Document_Review table
<=4.0 validation is needed

### LINKED ISSUE #14: Add Comment attribute in the Document_Review table
this attribute could be empty when a reviewer do the review

### LINKED ISSUE #16: Calculate weighted average score of a resume
Weighted average score from different reviewers of a user can be calculated by adding a function calculateScore that can calculate the weighted average score of the above three aspects given by reviewers

### PR DESCRIPTION
**Pull request by Lucybean-hi**
comment
skillScore
experienceScore
gpa
averageScore function

### CODE REVIEW

**Comment by Yuqi07** [comment]
Great job! I am just wondering if we can change setGPA / getGPA to setAcademicScore / getAcademicScore. It's a bit wordy, but the consistency with frontend naming convention is better.

**Comment by erica-w-fu** [comment]
I can change my wording to setGPA too. I think that would match the new implementation (entering the GPA rather than rating on a scale) better as well!

**Comment by erica-w-fu** [comment]
Also, what is the comment for? I noticed there were a lot of getComment() and setComment() in your changes

**Comment by juliawgraham** [comment]
Everything looks good to me, but I just have the same question as @erica-w-fu

**Comment by juliawgraham** [comment]
Also, this PR is going to main, which should not be the case. It seems like these changes are on the multiple-rating branch already (as this is in the 'integration branch', so I don't this this PR is necessary. Please feel free to open this PR back up if I am missing something.

**Review by Yuqi07 — COMMENTED** [review]
image

The box can correctly display the previously selected score. That's awesome!
